Makefile.core.mk

```diff
@@ -56,39 +125,151 @@ googleapis_protos = \
 	google/type/latlng.proto \
 	google/type/money.proto \
 	google/type/postal_address.proto \
-	google/type/timeofday.proto \
```

Comment:
surprised this worked.  does it work without it?

Reply:
What's surprising about it?